src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Generated/TextAnalytics/TextAnalyticsClientExtensions.cs

```diff
@@ -135,13 +136,172 @@ public static async Task<KeyPhraseBatchResult> KeyPhrasesAsync(this ITextAnalyti
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<SentimentBatchResult> SentimentAsync(this ITextAnalyticsClient operations, bool? showStats = default(bool?), MultiLanguageBatchInput multiLanguageBatchInput = default(MultiLanguageBatchInput), CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is a breaking change and needs a major version bump for the nuget package.

Reply:
There is no major change in Text Analytics API. We would like to avoid auto-generate and directly update the sdk here. Should we use version number <3.1.0>?

Reply:
Opt for 3.1.0 version bump